### PR TITLE
ipa: improve the introduction

### PR DESCRIPTION
**Pull request by pbrezina**

### CODE REVIEW

File: src/docs/ipa/ipa-introduction.rst [hunk 1]
```diff
@@ -1,41 +1,61 @@
-Introduction to FreeIPA Integration
-###################################
+Introduction to FreeIPA
+#######################
 
-SSSD is a key client-side component of the FreeIPA (also known as Red Hat Identity Management in RHEL, or simply **IPA**) architecture. SSSD is running on both IPA servers and IPA clients and is used to
+`FreeIPA`_ is an open source product that combines multiple technologies and
+protocols into single complex identity management solution. It provides much
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:5]: s/into single/into a single/

File: src/docs/ipa/ipa-introduction.rst [hunk 2]
```diff
@@ -1,41 +1,61 @@
...
+richer experience when compared to native LDAP solutions such as:
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:6]: s/provides much richer/provides a much richer/

Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:6]: s/such as/including features such as/

File: src/docs/ipa/ipa-introduction.rst [hunk 3]
```diff
@@ -1,41 +1,61 @@
...
-   * Automount maps
-   * SELinux user maps
-   * Netgroups
+.. note::
 
-* Perform authentication with IPA services like IPA's embedded Kerberos server/KDC
-* Allow IPA user authentication though ``pam_sss`` PAM module
-* Automated Client Service discovery and failover
-* Dynamic DNS Updates
+    SSSD is the main FreeIPA client therefore it provides the full experience
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:19]: s/provides/supports/

File: src/docs/ipa/ipa-introduction.rst [hunk 4]
```diff
@@ -1,41 +1,61 @@
...
+    and always provides every new feature at the same time when it becomes
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:20]: s/provides/supports/

File: src/docs/ipa/ipa-introduction.rst [hunk 5]
```diff
@@ -1,41 +1,61 @@
...
 
-* `FreeIPA Client <https://www.freeipa.org/page/Client>`_
+Active Directory Integration
+****************************
 
-Or on the terminal to read about SSSD's IPA provider
+One of the main FreeIPA features is its ability to seamlessly integrate with
+`Active Directory`_. The integration is achieved through creating a trust with
+existing Active Directory domains. Users and groups from trusted domains are
+then available on FreeIPA enrolled host (which also means that Active Directory
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:40]: s/host/hosts/

File: src/docs/ipa/ipa-introduction.rst [hunk 6]
```diff
@@ -1,41 +1,61 @@
...
+#######################
 
-SSSD is a key client-side component of the FreeIPA (also known as Red Hat Identity Management in RHEL, or simply **IPA**) architecture. SSSD is running on both IPA servers and IPA clients and is used to
+`FreeIPA`_ is an open source product that combines multiple technologies and
+protocols into single complex identity management solution. It provides much
+richer experience when compared to native LDAP solutions such as:
 
-* Retrieve and cache data stored in IPA LDAP database, including
+* Support for two factor and smartcard-based authentication
+* Host-Based Access Controllers (HBAC)
```
Comment (justin-stephenson) [inline src/docs/ipa/ipa-introduction.rst:9]: s/Controllers/Control/ ?

**Review by justin-stephenson — CHANGES_REQUESTED** [review]
Minor changes added, overall LGTM.

**Comment by pbrezina** [comment]
Thank you. Please, see new patches.

**Comment by justin-stephenson** [comment]
Ack.

**Pull request merged**